src/plugins/telemetryTable/components/table.vue

```diff
@@ -847,6 +851,25 @@ export default {
 
             this.isShowingMarkedRowsOnly = false;
         },
+        boundsChanged(_bounds, isTick) {
+            if (isTick) {
+                return;
+            }
+
+            this.userBoundsChanged();
+        },
+        userBoundsChanged() {
```

Comment:
i think this can all be replaced by `unpause(true)`. Took me a while to figure out what `pausedByButton` represents, but i think its main(only) purpose is if someone unmarks the last marked row, we want the table to remain paused if the user paused explicitly originally by clicking the pause button, as opposed to pausing the table by marking a row.